utilities/blob_db/blob_db_impl.h

```diff
@@ -373,7 +376,7 @@ class BlobDBImpl : public BlobDB {
 
   // all the blob files which are currently being appended to based
   // on variety of incoming TTL's
-  std::set<std::shared_ptr<BlobFile>, blobf_compare_ttl> open_ttl_files_;
```

Comment:
Almost make a mistake.. The TTL version of comparator is also needed here. I'm keeping it now.

Reply:
Oh, ok. Wow, at least updating the name to be consistent with the logic caught a sleeper-bug.

Reply:
yup.. not sure why tests not catching it.